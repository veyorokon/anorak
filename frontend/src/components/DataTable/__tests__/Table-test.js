/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import React from 'react';
import { mount, shallow } from 'enzyme';
import { Table } from '../';

describe('DataTable.Table', () => {
  it('should render', () => {
    const wrapper = mount(<Table className="custom-class" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should support disable zebra stripe', () => {
    const wrapper = shallow(<Table zebra={false} />);
    expect(wrapper.hasClass('bx--data-table')).toBe(true);
    expect(wrapper.hasClass('bx--data-table--zebra')).toBe(false);
  });
});
