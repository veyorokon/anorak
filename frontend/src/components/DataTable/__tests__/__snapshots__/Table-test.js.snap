// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.Table should render 1`] = `
<Table
  className="custom-class"
  short={false}
  shouldShowBorder={true}
  zebra={true}
>
  <table
    className="bx--data-table custom-class bx--data-table--zebra"
  />
</Table>
`;
